student/index.js

```diff
@@ -3,19 +3,24 @@
 // return the string
 function returnTheString (string) {
   // your code here
+  return(string);
 }
 
 // Exercise 1
 // Given a string,
 // turn it into an array of the letters in the string
 function splitTheString (string) {
   // your code here
+  const arrayOfLetters =[...string];
+  return arrayOfLetters;
 }
 
 // Exercise 2
 // Given an object,
 // return a shallow copy of the object
 function shallowCopyObject (object) {
+  const NewObject={...object};
```

Comment:
This solution is correct. Just a very minor comment: normal variable names should always start with a small letter In JavaScript. Variable names with a capital letter are usually reserved for special cases like classes or in React also react components. 
So this line is absolutely correct(!), this is just a common convention :)